tajo-catalog/tajo-catalog-client/src/main/proto/CatalogProtocol.proto

```diff
@@ -24,53 +24,128 @@ option java_generate_equals_and_hash = true;
 import "CatalogProtos.proto";
 import "PrimitiveProtos.proto";
 
+message GetTablespacesResponse {
+  required ReturnState state = 1;
+  repeated TablespaceProto tablespace = 2;
+}
+
+message GetTablespaceResponse {
+  required ReturnState state = 1;
+  optional TablespaceProto tablespace = 2;
+}
+
+message GetDatabasesResponse {
+  required ReturnState state = 1;
+  repeated DatabaseProto database = 2;
+}
+
+message TableDescResponse {
+  required ReturnState state = 1;
+  optional TableDescProto table = 2;
+}
+
+message GetTablesResponse {
+  required ReturnState state = 1;
+  repeated TableDescriptorProto table = 2;
+}
+
+message FunctionDescResponse {
```

Comment:
My intension was to merge it into FunctionResponse in ClientProtos.proto because duplicate proto type is used in multiple places. But, I finished the merge yet. I'll merge them in this patch.